src/main/resources/templates/score/list.html

```diff
@@ -6,6 +6,11 @@
 <div th:insert="fragments/organizerHeader :: header(score)"></div>
 <div class="container">
     <div class="row">
+        <div class="col-sm-12">
+            <a href="#" class="btn btn-primary" th:href="@{/game/score/export}" target="_blank">
+                Exportovat skóre do PDF
```

Comment:
Když je tohle tlačítko na stránce se sckore, co to přejmenovat jednoduššne na Export do PDF? Nebo jen PDF?
Dávám k diskuzi.

Reply:
Export do PDF zní dobře :)